core/api/views/replenishment.py

```diff
@@ -397,6 +419,36 @@ def get(self, request, *args, **kwargs):
             .order_by("name")
         ]
 
+        # TODO: I think this might return duplicates if a country has multiple
```

Comment:
I think you could also do the change here. :)

Reply:
Done :D